README.md

```diff
@@ -226,13 +226,13 @@ If the wrong version of a translation is being used, ensure a release with the l
 
 
 ## Limitations
-
+- Wrapping `Context` with Phrase can lead to issues with `WebView`. Consider to use `Context.getPhrase*()`/`Resources.getPhrase*()` extension functions to get translations via Phrase. 
```

Comment:
Do you think we could describe just a little in which cases in could cause issues?